main.gd

```diff
@@ -46,7 +46,7 @@ func _ready() -> void:
 	tree.set_column_title(2, "Render GPU")
 	tree.set_column_title(3, "Idle")
 	tree.set_column_title(4, "Physics")
-	tree.set_column_title(5, "Wall Clock Time")
+	tree.set_column_title(5, "Startup Time")
```

Comment:
Regarding measuring wall clock time, this was designed for GDScript tests which run a fixed amount of iterations.

Reply:
It essentially changes the time measurement to exclude the fixed 5 seconds spent running the scene (since that's always... well, 5 seconds), so GDScript tests would be part of the "Startup Time" and still get tracked like before.

"Startup Time" maybe isn't the best name for this.

Reply:
Maybe we should just rename the title to "Main Thread Time", and explain what this is used for depending on the benchmark type in the README.

Reply:
Done